server/models/book.go

```diff
@@ -0,0 +1,8 @@
+package models
```

Comment:
I think this could go under the `server/database` under the database function if it's used as the type that's fetched from the db.